UI/auth-oauth.cpp

```diff
@@ -64,6 +64,8 @@ OAuthLogin::OAuthLogin(QWidget *parent, const std::string &url, bool token)
 	QVBoxLayout *topLayout = new QVBoxLayout(this);
 	topLayout->addWidget(cefWidget);
 	topLayout->addLayout(bottomLayout);
+#else
+	(void)url; // Avoid compiler warining about unused variable on windows
```

Comment:
obs has a macros UNUSED_PARAMETER(url); 
if it available in UI code better to use it here too.

Reply:
Thanks, used this.